spec/controllers/styled_forecasts_controller_spec.rb

```diff
@@ -0,0 +1,69 @@
+RSpec.describe StyledForecastsController do
+  around do |example|
+    env_vars = {
+      CERC_API_HOST_URL: "https://cerc.example.com",
+      CERC_API_KEY: "SECRET-API-KEY"
+    }
+    ClimateControl.modify(env_vars) { example.run }
+  end
+
+  describe "GET :show" do
+    let(:json) do
+      <<~JSON
```

Comment:
Is there a reason we're defining this as JSON only to then parse it into a hash? A simpler alternative would be to define it as a hash from the start and skip the conversion step.

Reply:
yeah, fair point. This is Ed's work that I copied over from the "v1" controller spec, but I would guess the idea is to make it as clear as possible that we're generating the forecasts from an API response? But if so, that is documented in other tests. I'll skip the conversion as you suggest 👍